policies/P.C.03.rego

```diff
@@ -0,0 +1,14 @@
+package containers_securitycontext_capabilities_add_index_sys_admim
+
+import data.lib.kubernetes
+
+# https://kubesec.io/basics/containers-securitycontext-capabilities-add-index-sys-admin/
+violation[msg] {
+	kubernetes.containers[container]
+	added_capability(container, "CAP_SYS_ADMIN")
```

Comment:
what if there will be multiple capabilities be allowed?

Reply:
This is only disallowing one singular capability, although I might end up adding more in the future for now the kubesec recommend this one be blocked.